### ISSUE TITLE
Migrate the website to PkgPage.jl

### ISSUE THREAD

**Issue by ViralBShah** [FluxML/fluxml.github.io#64]

**Comment by logankilpatrick** [comment]
@ViralBShah Do you still think this is necessary or given the updates by @lilianabs are we good to close this?

**Comment by ViralBShah** [comment]
Not necessary - just nice to use Franklin to make things easy to edit. Jekyll is almost impossible nowadays.

**Comment by logankilpatrick** [comment]
Agreed, feel free to close if you think we are in a good place now.

**Comment by ViralBShah** [comment]
We're still on jekyll from what I can see. So let's leave it open.
